browser/src/Services/BufferUpdates.ts

```diff
@@ -52,5 +52,15 @@ export class BufferUpdates {
                 this._pluginManager.notifyBufferUpdate(args, bufferLines)
             }
         })
+        
+        this._neovimInstance.on("buffer-update-incremental", (args: Oni.EventContext, bufferLine: string, lineNumber: number) => {
+            // const lastLine = args.line
+            // this._lastArgs = args
+            // this._lastBufferLines = bufferLine
+            // this._lastBufferVersion = args.version
+
+            // If we can send incremental updates, and the line hasn't changed, just send the incremental change
```

Comment:
@extr0py:  So it's ok to just update lastBufferLines with just the single line that gets sent over? It seemed like that was always updated with the whole buffer, not just a single line.